packages/sdk/src/client.ts

```diff
@@ -270,13 +277,17 @@ export function SoundClient({
 
     switch (mintSchedule.mintType) {
       case 'RangeEdition': {
-        return RangeEditionMinter__factory.connect(mintSchedule.minterAddress, signer).mint(
-          mintSchedule.editionAddress,
-          mintSchedule.mintId,
-          quantity,
-          affiliate,
-          txnOverrides,
-        )
+        const mintArgs = [mintSchedule.editionAddress, mintSchedule.mintId, quantity, affiliate, txnOverrides] as const
+
+        const gasEstimate = await RangeEditionMinter__factory.connect(
```

Comment:
If a gas estimate is set, don’t make this call at all.
If this call fails, then catch and don’t throw, fall back to auto estimate